pkg/virt-handler/vm.go

```diff
@@ -2667,7 +2671,7 @@ func (d *VirtualMachineController) handleTargetMigrationProxy(vmi *v1.VirtualMac
 	baseDir := fmt.Sprintf(filepath.Join(d.virtLauncherFSRunDirPattern, "kubevirt"), res.Pid())
 	migrationTargetSockets = append(migrationTargetSockets, socketFile)
 
-	isBlockMigration := vmi.Status.MigrationMethod == v1.BlockMigration
+	isBlockMigration := vmi.Status.MigrationMethod == v1.BlockMigration || len(vmi.Status.MigratedVolumes) > 0
```

Comment:
It will be great to have a function that performs this in a centralized way. 
Like `vmi.IsMigratable()` we can have:
```go
func (v *VirtualMachineInstance) IsBlockMigration() bool {
	return v.Status.MigrationMethod == BlockMigration || len(v.Status.MigratedVolumes) > 0
}
```
and use it, instead of replicate the check.

Reply:
@fossedihelm I'm not quite sure where the unit test should be. IMO, we are missing some unit tests for the migration parameters for `live-migration-source.go`

Reply:
Maybe somewhere here? https://github.com/kubevirt/kubevirt/blob/fba31d300a12e7c1973a965dd7f5cccda6e83cb2/pkg/virt-handler/vm_test.go#L2229

Reply:
Ok, let's close the gap in a followup PR. Thanks